docs/src/blockchain-development/calling_contracts.md

```diff
@@ -0,0 +1,127 @@
+# Calling Contracts
+
+End users and smart contracts typically need to communicate with each other.
+
+The FuelVM achieves end user and inter contract communication primarily with the [`call`](https://github.com/FuelLabs/fuel-specs/blob/master/specs/vm/opcodes.md#call-call-contract) opcode.
+
+Sway provides a nice way manage calls with its `abi` system.
```

Comment:
```suggestion
Smart contracts can be _called_ by other contracts or scripts. In the FuelVM, this is done primarily with the [`call`](https://github.com/FuelLabs/fuel-specs/blob/master/specs/vm/opcodes.md#call-call-contract) instruction.

Sway provides a nice way manage callable interfaces with its `abi` system.
```

Reply:
Added, but couldn't commit change due to other changes before.